docker/sagemaker/Dockerfile.sm

```diff
@@ -1,22 +1,11 @@
 # Docker file for building a docker image for running GraphStorm code on Amazon SageMaker
-# Note: Distributed graph partition will use another docker image which will come soon.
-
 ARG DEVICE=gpu
-ARG DGL_VERSION=2.3.0
+ARG SOURCE
+ARG USE_PARMETIS=false
```

Comment:
Do we already support the parmetis partition on sagemaker or is it just a placeholder?

Reply:
Placeholder here, allows us to have same arguments for both local and sagemaker builds.